scripts/hashlint.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
```

Comment:
Could you add this script to the Python lint workflow?

Reply:
I thought it'd be caught by `**.py` but I can add this one explicitly if needed

Reply:
Just checked that workflow file... we trigger on **py but only lint `roles/*/*py`... so that's another thing :(

Reply:
I'll clean that up while I'm in there.